src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -530,7 +530,7 @@ protected function initializeContainer(): void
 
         $buildDir = $this->container->getParameter('kernel.build_dir');
         $cacheDir = $this->container->getParameter('kernel.cache_dir');
-        $preload = $this instanceof WarmableInterface ? (array) $this->warmUp($cacheDir, $buildDir) : [];
+        $preload = $this instanceof WarmableInterface ? $this->warmUp($cacheDir, $buildDir) : [];
```

Comment:
Is the array cast on line 542 also removable ? 
I'm not sure PHP while do it automatically.

Reply:
Indeed :+1: